recipe/meta.yaml

```diff
@@ -29,28 +29,30 @@ requirements:
     - dask
     - entrypoints
     - fsspec
-    - fiona ==1.8.22
-    - gdal >=3.1
     - geopandas >=0.10
-    - netcdf4
-    - numpy >=1.20
+    - mercantile
     - numba
+    - numpy >=1.20
+    - netcdf4
     - openpyxl
```

Comment:
openpyxl is officially also an optional dependency in our pyproject.toml. I think we could leave it out here.